process/tags/tagsCount.go

```diff
@@ -0,0 +1,115 @@
+package tags
+
+import (
+	"bytes"
+	"encoding/json"
+
+	"github.com/ElrondNetwork/elastic-indexer-go/data"
+)
+
+const (
+	tagsKey = "tags"
+)
+
+// CountTags defines what a TagCount handler should be able to do
+type CountTags interface {
+	Serialize() ([]*bytes.Buffer, error)
+	ParseTagsFromDB(response map[string]interface{}) error
+	ExtractTagsFromAttributes(attributes *data.Attributes)
+	GetTags() []string
+	Len() int
+}
+
+type tagsCount struct {
+	tags map[string]int
+}
+
+// NewTagsCount will create a new instance of tagsCount
+func NewTagsCount() CountTags {
+	return &tagsCount{
+		tags: make(map[string]int),
+	}
+}
+
+func (tc *tagsCount) Len() int {
+	return len(tc.tags)
+}
+
+func (tc *tagsCount) GetTags() []string {
+	tags := make([]string, 0, tc.Len())
+	for key := range tc.tags {
+		tags = append(tags, key)
+	}
+
+	return tags
+}
+
+func (tc *tagsCount) ExtractTagsFromAttributes(attributes *data.Attributes) {
+	tagsC := make(map[string]int)
+
+	if attributes == nil {
+		return
+	}
+
+	for key, tags := range *attributes {
+		if key != tagsKey {
+			continue
+		}
+
+		for _, tag := range tags {
+			tagsC[tag] = 1
```

Comment:
because you set 1 as a value in tagsC I think we can simplify this:
```GO
for key, tags := range *attributes {
		if key != tagsKey {
			continue
		}

		for _, tag := range tags {
                       tc.tags[key]++
                }
}
```

of course the second for can be extracted in a new function. What do you think?


the portion beyond L65 is no longer necessary.

Reply:
In this case, I don't want increment because we have the next scenario: if someone will create a NFT and will put a tag multiple times  I want to count only one time.

Reply:
Implemented your solution with a small modification.
```

	for key, tags := range *attributes {
		if key != tagsKey {
			continue
		}

		newTags := removeDuplicatedTags(tags)
		for _, tag := range newTags {
			tc.tags[tag]++
		}
	}
```
Added method `removeDuplicatedTags`

Reply:
right, seems fine :+1: